src/components/common/HorizontalScroll/HorizontalScroll.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import { Meta, StoryObj } from '@storybook/react';
+import HorizontalScroll from './HorizontalScroll';
+import { Card, Image } from '..';
+
+const meta: Meta<typeof HorizontalScroll> = {
+  title: 'Component/HorizontalScroll',
+  component: HorizontalScroll
+};
+
+export default meta;
+type Story = StoryObj<typeof HorizontalScroll>;
+
+export const ExampleCardScroll: Story = {
+  args: {
+    className: 'w-80'
+  },
+  render: (args) => (
+    <HorizontalScroll className={args.className}>
```

Comment:
p5: 중요하지는 않지만 동일하게 적용되서 제안드립니다..!
```suggestion
    <HorizontalScroll {...args}>
```